### PR TITLE
Remove charts

### PR DESCRIPTION
**Pull request by ostromart**
This completes the work to remove the local charts from operator. We are now using a installer.sha to specify the target in istio/installer. 
Right now we are pulling installer charts into data/charts for building. This is necessary because the embedded, self-compiling generator cannot reference external dirs easily. 
Once https://github.com/istio/operator/pull/180 goes in, we will be able to remove this limitation.
https://github.com/istio/istio/issues/16252 tracks this.

### CODE REVIEW

**Comment by richardwxn** [comment]
Please update the README as well.

Can we default to update from the latest master if no sha provided

File: data/profiles/default.yaml [hunk 1]
```diff
@@ -80,7 +79,7 @@ spec:
             resources:
               requests:
                 cpu: 100m
-                memory: 128Mi
+                memory: 1024Mi
```
Comment (morvencao) [inline data/profiles/default.yaml]: Is this a bug in upstream? Why requested memory size is bigger than limited memory size?
Should we set it as default resources settings for pilot https://github.com/istio/installer/blob/master/istio-control/istio-discovery/values.yaml#L17-L20

Comment (ostromart) [reply data/profiles/default.yaml]: This is one of several outstanding diffs that will still need to be addressed. I propose we merge this as is to progress, and start iterating on eliminating the remaining changes incrementally - wdyt?

**Comment by ostromart** [comment]
@richardwxn what do you mean regarding the README?

File: data/profiles/default.yaml (hunk 1 above)
Comment (morvencao) [reply data/profiles/default.yaml]: OK for me.

**Comment by ostromart** [comment]
@richardwxn, you can already say "master" or branch name instead of sha - I updated the usage.

**Comment by richardwxn** [comment]
> @richardwxn what do you mean regarding the README?

oh I mean there are some places in README still referencing the usage of compiled in chart. But we can fix that later

**Pull request merged**